keras_hub/src/models/deeplab_v3/deeplab_v3_presets.py

```diff
@@ -1,4 +1,17 @@
 """DeepLabV3 preset configurations."""
 
-# TODO https://github.com/keras-team/keras-hub/issues/1896,
-backbone_presets = {}
+backbone_presets = {
+    "deeplabv3_plus_resnet50_pascalvoc": {
+        "metadata": {
+            "description": (
+                "DeepLabV3+ model with ResNet50 as image encoder and trained on "
+                "augmented Pascal VOC dataset by Semantic Boundaries Dataset(SBD)."
```

Comment:
add accuracy and mIOU results in the description

Reply:
Done

Reply:
Hi Sachin! looks like you might not have committed the changes.